### ISSUE TITLE
Bug: admin products_to_categories SQL 1064 You have an error in your SQL syntax

### ISSUE THREAD

**Issue by torvista** [zencart/zencart#3893]
```
// @TODO this is a pretty elaborate query for getting a single products_id in order to do a redirect!
    $sql = "SELECT ptc.*
            FROM " . TABLE_PRODUCTS_TO_CATEGORIES . " ptc
            LEFT JOIN " . TABLE_PRODUCTS_DESCRIPTION . " pd USING (products_id)
            JOIN " . TABLE_PRODUCTS . " p USING (products_id)
            AND pd.language_id = " . (int)$_SESSION['languages_id'] . "
            WHERE ptc.categories_id = " . $current_category_id . "
            ORDER BY p.products_model"; // Order By determines which product is pre-selected in the list when a new category is viewed
```
1) "Interestingly" this passes in ZC157, but breaks in ZC158: the WHERE should be before the AND ("1064 You have an error in your SQL syntax;" etc.)
2) Once corrected, when the product list is displayed, a product is preselected. If another is selected, the page refreshes but back to the original product, NOT the selected product.
This is working in ZC157 and since there are a pile of changes between 157 and 158, I'll have to pass on this.

**Comment by scottcwilson** [comment]
1.5.7: admin/products_to_categories.php line 209
1.5.8: admin/products_to_categories.php line 60

**Comment by scottcwilson** [comment]
This might have been fixed in e55a2babd64848ece57fb739342cd7e3373c9393

**Comment by drbyte** [comment]
Appears to be resolved.
